lib/citgm.js

```diff
@@ -98,15 +123,15 @@ Tester.prototype.run = function() {
 
   async.waterfall([
     init.bind(null, this),
-    find.bind(null, 'node'),
-    find.bind(null, 'npm'),
+    findNode.bind(null),
```

Comment:
I think we can drop the bind altogether here can't we?

Reply:
Yeah, good catch!